src/models/Level.ts

```diff
@@ -11,6 +11,7 @@ const Level = types.model({
 	entities: types.array(Entity),
 }).actions((self) => ({
 	set2StarsTime(ms: number): void {
+		ms *= 1000;
```

Comment:
These aren't ms :wink:
So please rename the variable to `sec`.

Actually, wait. Can you move this conversion to `on2StarsChange`? https://github.com/getkey/ble/pull/27/files#diff-12ab7adb829f8fa69daa0838e477471097f40be2af1f11133a6b4accc9f7fd96R47

It would be more consistent. When you grab the timings [here](https://github.com/getkey/ble/pull/27/files#diff-12ab7adb829f8fa69daa0838e477471097f40be2af1f11133a6b4accc9f7fd96R44) **from the model**, you _get_ milliseconds.
So when you _set_ the timings **in the model** here, by symmetry, you should set milliseconds as well. (unless this method was explicitly called `set2StarsTimeInSeconds` or something, but I don't think it's nice to make this super verbose)